testsuite/manualmode/src/test/java/org/jboss/as/test/manualmode/management/cli/ReloadOpsTestCase.java

```diff
@@ -91,7 +77,7 @@ public void testWriteAttribvuteWithReload() throws Exception {
         result = cli.readAllAsOpResult();
         assertTrue(result.isIsOutcomeSuccess());
         checkResponseHeadersForProcessState(result);
-        reloadServer(TIMEOUT);
+        container.reload(TIMEOUT);
```

Comment:
I removed this as it was unifying the code, but yeah it does make sense to use the CLI for this.